lib/clone.js

```diff
@@ -76,11 +76,6 @@ function clone(pkg, src, dst, credentials, xform, ready) {
         data.pipe(concat(attachReadme))
           .on('error', onerror)
       }
-
-      function attachReadme(data) {
-        metadata.readme = data.toString()
-        parse.end()
```

Comment:
Curious: what about `parse.pause()` and calling `onend` here explicitly (since we don't care about the rest of the parse)?